README.org

```diff
@@ -81,17 +94,14 @@ If you have more specific needs (a front-end db, etc), instead of an atom, you c
 
 
 
-#+BEGIN_SRC clojure
-(rfc/render-application test-form {:READ (partial get-in @my-atom)
-                                   :UPDATE (partial swap! my-atom update-in)})
-#+END_SRC
+
 
 
 ** Element and Attribute Reference
 
 *** universal keys
 **** =:type=
-indicates which element to create.  Can be any of: =:select= =:multi-table= =:textarea= =:togglebox= =:checkbox= =:file= =:hidden=
+indicates which element to create.  Can be any of: =:select=, =:radio=, =:multi-table=, =:textarea=, =:togglebox=, =:checkbox=, =:file=, =:hidden=
 
 **** =:subtext=
 Explanatory or parenthetical text, appearing in smaller font under =:label=
```

Comment:
Not accurate. Rather than "appearing in smaller font" should be something like "appearing in <div.subtitile>" or whatever it is, since font size is up to the styling.